src/App.js

```diff
@@ -1,26 +1,17 @@
-import React from 'react';
-import logo from './logo.svg';
-import './App.css';
+import React from 'react'
+import { Route, Switch } from 'react-router-dom'
+import GlobalStyles from './GlobalStyles'
+import PetProfile from './components/PetProfile'
 
-function App() {
+export default function App() {
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
+    <>
+      <GlobalStyles />
+      <Switch>
+        <Route exact path="/pet-profile">
+          <PetProfile />
+        </Route>
```

Comment:
Is the pet profile currently the only page in your app? In that case you probably wouldn't need a Router just yet. But you probably included it for future stories?

Reply:
Exactly. I was thinking ahead, but I probably should have waited until my next Sprint to add the router.